code/modules/mob/living/simple_animal/abnormality/aleph/distortedform.dm

```diff
@@ -1433,6 +1436,56 @@
 	REMOVE_TRAIT(H, TRAIT_MUTE, TIMESTOP_TRAIT)
 	H.remove_atom_colour(TEMPORARY_COLOUR_PRIORITY)
 
+
+//You’re Bald...
+/mob/living/simple_animal/hostile/abnormality/distortedform/proc/ChangeBald()//man roleplayers are going to hate this one
+	transform_cooldown = transform_cooldown_time_short + world.time
+	name = "You’re Bald..."
+	desc = "A helpful sphere, you think."
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "bald1"
+	icon_living = "bald1"
+	pixel_x = 0
+	base_pixel_x = 0
+	pixel_y = 0
+	base_pixel_y = 0
+	can_move = FALSE
+	can_attack = FALSE
+	can_act = FALSE //we stay transformed until the skill finishes firing
+	addtimer(CALLBACK(src, PROC_REF(BaldBlast), FALSE), 5)
+
+
+/mob/living/simple_animal/hostile/abnormality/distortedform/proc/BaldBlast(attack_chain)
+	if(!attack_chain)
+		icon_state = "bald3"
+		src.set_light(12, 12, "FFFFFF", TRUE)
+		playsound(get_turf(src), 'sound/abnormalities/sphinx/stone_ready.ogg', 50, 0, 5)
+	SLEEP_CHECK_DEATH(9)
+	if(attack_chain)
+		for(var/mob/living/L in viewers(12, src))
+			if(!ishuman(L))
+				continue
+			var/mob/living/carbon/human/H = L
+			if(!H.is_blind() && is_A_facing_B(H,src))
+				if(!HAS_TRAIT(H, TRAIT_BALD))
+					H.emote("scream")
+					H.Stun(20)
+					H.Paralyze(20)
+					H.Knockdown(200)
+					H.adjust_blindness(2)
+					to_chat(H, span_notice("You feel awesome?"))
+					ADD_TRAIT(H, TRAIT_BALD, "ABNORMALITY_BALD")
+					H.hairstyle = "Bald"
+					H.update_hair()
+					H.apply_damage(100, WHITE_DAMAGE, null, H.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
+					if(H.sanity_lost) // They can't deal with being bald
+						H.dust()
```

Comment:
My only gripe with this is that it does NOTHING to bald people. C'mon, crabby! This is supposed to be a superboss! There are no free lunches during this fight!